src/components/lib/Header.js

```diff
@@ -57,7 +57,7 @@ const HeaderTemplate = ({
                   <NavLink {...item} activeClassName="active" />
                 </li>
               ))}
-              <li><a className="mint-button" href={dappLink.url} target="_blank" rel="noopener noreferrer">{dappLink.label}</a></li>
+              <li><a className="mint-button" href={dappLink.url} target="_blank" rel="noopener noreferrer"><span>{dappLink.label}</span></a></li>
```

Comment:
You know I wrinkled my nose at this one.

![wrinkling nose gif](https://media.giphy.com/media/JRU66SXpkqlvq/giphy.gif)

But I get it :wink: